spec/git_helper_spec.rb

```diff
@@ -90,4 +91,58 @@
       Fastlane::Helper::GitHelper.commit(message: @message, push: true)
     end
   end
+
+  describe '#is_ignored?' do
+    let(:path) { 'dummy.txt' }
+
+    it 'returns false when the path is not ignored' do
+      setup_git_repo(
+        dummy_file_path: path,
+        in_gitignore: false
+      )
+      expect(Fastlane::Helper::GitHelper.is_ignored?(path: path)).to be false
+    end
+
+    context 'when the path is in the .gitignore' do
+      # rubocop:disable RSpec/ExampleLength
+      it 'returns true when the .gitignore has uncommitted changes' do
+        # For some reason, I was expecting the underlying `git check-ignore`
+        # command to fail in this case, but I'm clearly wrong.
+        #
+        # I think there's value in keeping this behavior explicity documented
+        # and verified here. – Gio
+        setup_git_repo(
+          dummy_file_path: path,
+          in_gitignore: true,
+          gitignore_committed: false
+        )
+        expect(Fastlane::Helper::GitHelper.is_ignored?(path: path)).to be true
+      end
+
+      it 'returns true when the .gitignore has no uncommitted changes' do
+        setup_git_repo(
+          dummy_file_path: path,
+          in_gitignore: true,
+          gitignore_committed: true
+        )
+        expect(Fastlane::Helper::GitHelper.is_ignored?(path: path)).to be true
+      end
+      # rubocop:enable RSpec/ExampleLength
+    end
+  end
+end
+# rubocop:enable Metrics/BlockLength
```

Comment:
(don't forget to remove this one if you do remove the disable after my previous comment)